client/src/components/PortfolioFilters.tsx

```diff
@@ -255,67 +257,139 @@ export const PortfolioFilters = React.memo(
-              <span className="active-filter-count">{activeFilters}</span>
-            )}
-            {isOpen ? <CloseIcon className="closeIcon" /> : <ChevronIcon className="chevronIcon" />}
-          </summary>
-          <div className="dropdown-container scroll-gradient mobile-wrapper-dropdown">
-            {children}
-            {!!activeFilters && (
-              <button onClick={() => setIsOpen(!isOpen)} className="button is-primary">
-                <Trans>View Results</Trans>
-                {resultsCount != null && <span className="view-results-count">{resultsCount}</span>}
-              </button>
-            )}
+          <div className="filters">
+            <details
+              className={classnames("filter filter-accordion filters-mobile-wrapper", {
+                active: !!numActiveFilters,
+              })}
+              open={isOpen}
+            >
+              <summary
+                onClick={(e) => {
+                  e.preventDefault();
+                  setIsOpen(!isOpen);
+                }}
+              >
+                <Trans>Filters</Trans>
+                {!isOpen && !!numActiveFilters && (
+                  <span className="active-filter-count">{numActiveFilters}</span>
+                )}
+                {isOpen ? (
+                  <CloseIcon className="closeIcon" />
+                ) : (
+                  <ChevronIcon className="chevronIcon" />
+                )}
+              </summary>
+              <div className="dropdown-container scroll-gradient mobile-wrapper-dropdown">
+                {children}
+                {!!activeFilters && (
+                  <button onClick={() => setIsOpen(!isOpen)} className="button is-primary">
+                    <Trans>View Results</Trans>
+                    {resultsCount != null && (
+                      <span className="view-results-count">{resultsCount}</span>
+                    )}
+                  </button>
+                )}
+                {resultsCount === 0 ? ZeroResultsAlert : <></>}
+              </div>
+            </details>
           </div>
-        </details>
-      </div>
-    </FocusTrap>
+        </FocusTrap>
+      )}
+      {activeFilters.rsunitslatestActive && (!isOpen || !isMobile) && resultsCount ? (
+        RsUnitsToastAlert
+      ) : (
+        <></>
+      )}
+      {activeFilters.ownernamesActive && (!isOpen || !isMobile) && resultsCount ? (
+        OwnernamesToastAlert
+      ) : (
+        <></>
+      )}
+    </>
   );
 };
 
-const OwnerInfoAlert = (
+const OwnernamesToastAlert = (
+  <Alert
+    className="ownernames-toast-alert filter-toast-alert"
+    type="info"
+    variant="secondary"
+    closeType="none"
+    role="status"
+  >
+    <Trans>
+      Expand the the Owner/Manager column in Table view to see all contacts associated with that
+      building.
+    </Trans>
+  </Alert>
+);
+
+const RsUnitsToastAlert = (
+  <Alert
+    className="ownernames-toast-alert filter-toast-alert"
```

Comment:
should the class for the RsUnits still be `ownernames-toast-alert` ? maybe this class could be removed altogether (here & in the above component) if it's not used in the CSS?